Adding new ppl profiles (#558)
usds/website: _people/1150_meg-peters.md

# People Carousel and Page (Edit this)
name: Meg Peters
pronouns: She/Her
alumni: false
cop: Design
city_of_origin: Reston, VA
previously_at: Library of Congress
image_name: meg-peters.jpg
quote: "My friends here inspire me. When the work demands 110% from us, we lean on each other until we get “all the things” done."

---

**What’s your background?**

As a kid, I loved drawing pictures and making things. I walked through our neighborhood, wagon full of crayons and coloring books in tow, asking other kids to color with me. In college, I spent a lot of time researching and writing for my design projects. Art school didn’t require this, but I believed that the design and content of printed materials should support each other. Early in my career, I designed for print, wrote advertisements, and created websites for small businesses (design, content, and code). As I started working on big websites for companies, nonprofits, and government, I liked the challenge of organizing information so people could easily find what they need. I’ve gone by many job titles. “Information Engineer” is the one that always makes me smile.

**What inspired you to join USDS?**

> I joined USDS to help solve big, messy problems that make it hard for people to apply for and get government benefits. I wanted to work on teams that designed _with_ the people who would use what we designed. At USDS, we “design with users, not for them,” and we bake this process into everything we build. To design a thing well, you must understand why and how people use the thing. Sometimes you have to do a lot of digging to figure that out and translate it into a design that works well.

Reviewer: ```suggestion
I joined USDS to help solve big, messy problems that make it hard for people to apply for and get government benefits. I wanted to work on teams that designed _with_ the people who would use what we designed. At USDS, we “design with users, not for them,” and we bake this process into everything we build. To design a thing well, you must understand why and how people use the thing. Sometimes you have to do a lot of digging to figure that out and translate it into a design that works.
```

**What has been your biggest challenge?**

> Creating momentum has probably been the biggest challenge. Sometimes it seems impossible to move something forward, even one inch. Almost nothing we work on is small or simple, so I’ve learned to design my interactions with busy stakeholders with the same care I put into designing websites and applications. When asking stakeholders for feedback, I try to focus conversations on one thing at a time. This makes it easier to get thoughtful, targeted feedback. And it helps keep everyone on the same page. The scale and complexity we’re dealing with are daunting. Thousands of people apply for government benefits every day, often using fragile systems and technology that have been layered on top of each other for decades.

Reviewer: ```suggestion
Creating momentum has probably been the biggest challenge. Sometimes it seems impossible to move something forward, even one inch. Almost nothing we work on is small or simple, so I’ve learned to design my interactions with busy stakeholders with the same care I put into designing websites and applications. When asking stakeholders for feedback, I try to focus conversations on one thing at a time. This makes it easier to get thoughtful, targeted feedback, and it helps keep everyone on the same page. The scale and complexity we’re dealing with is daunting. Thousands of people apply for government benefits every day, often using fragile systems and technology that have been layered on top of one another for decades.
```

**How does your work make an impact?**

> In three-and-a-half years at USDS, I’ve worked at IRS, Small Business Administration, U.S. Citizenship and Immigration Services, Social Security Administration, Department of Justice, and Department of Housing and Urban Development. At the Small Business Administration (SBA), my team worked with our agency partners and contractors to digitize the application and renewal process for programs that help disadvantaged small businesses get government contracts. By February 2019, more than 48,000 small businesses had applied to SBA’s government contracting programs, and these programs supported 537,000 jobs and an estimated $90 billion in business across the federal government.

Reviewer: ```suggestion
In three and a half years at USDS, I’ve worked at IRS, Small Business Administration, U.S. Citizenship and Immigration Services, Social Security Administration, Department of Justice, and Department of Housing and Urban Development. At the Small Business Administration (SBA), my team worked with our agency partners and contractors to digitize the application and renewal process for programs that help disadvantaged small businesses get government contracts. By February 2019, more than 48,000 small businesses had applied to SBA’s government contracting programs, and these programs supported 537,000 jobs and an estimated $90 billion in business across the federal government.
```

**What do you want to do after USDS?**

> I want to keep making it easier for people to apply for and get government benefits. Since I love working on websites, I plan to do more of that.

Reviewer: ```suggestion
I want to keep making it easier for people to apply for and receive government benefits. Since I love working on websites, I plan to do more of that.
```
